app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+
+    has_many :tweets
```

Comment:
If you get database errors while trying to destroy user accounts, this would be the issue. Users who have existing tweets would not be able to destroy their account.

I also noticed you have added validations for your User email and password attributes. Because we are using Devise for user authentication, Devise will handle validating those 2 fields for us 🙂. You can leave the name validation, though!